server/pages/index.js

```diff
@@ -342,13 +342,77 @@ function setUpLoggedInRoute( req, res, next ) {
 	}
 }
 
-function setUpRoute( req, res, next ) {
-	if ( req.cookies.wordpress_logged_in ) {
-		// the user is probably logged in
-		setUpLoggedInRoute( req, res, next );
-	} else {
-		setUpLoggedOutRoute( req, res, next );
+/**
+ * Sets up a Content Security Policy header
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+ * @param {Object} req Express request object
+ * @param {Object} res Express response object
+ * @param {Function} next a callback to call when done
+ */
+function setUpCSP( req, res, next ) {
+	if ( ! req.originalUrl.match( /^\/log-in\// ) ) {
```

Comment:
if this is for security, maybe we should be more specific? does it matter if we are too broad in capturing the path?

```js
import URL form 'url'

const originalUrl = new URL( req.originalUrl );
if ( ! /^\/log-in\//.test( originalUrl.pathname ) ) {

}
```

Reply:
It's just so we will use that on `/log-in/*` only pages for now.